ads/google/a4a/traffic-experiments.js

```diff
@@ -209,21 +208,13 @@ function selectRandomProperty(obj) {
  */
 export function randomlySelectUnsetPageExperiments(win, experiments) {
   win.pageExperimentBranches = win.pageExperimentBranches || {};
-  if (getMode(win).localDev) {
-    // In local dev mode, it can be difficult to configure AMP_CONFIG
-    // externally.  Default it here if necessary.
-    win.AMP_CONFIG = win.AMP_CONFIG || {};
```

Comment:
All the experiment flags are calculated once, no matter AMP_CONFIG exists or not.
https://github.com/ampproject/amphtml/blob/master/src/experiments.js#L134